### ISSUE TITLE
Browser becomes unresponsive when I change a pivot to a scatter plot

### ISSUE THREAD

**Issue by APolyakova** [atoti/atoti#243]
This issue can be reproduced using IFRS 9 notebook.

The recordset I have in a pivot is a big one - 15.000 points - the data is retrived in a sub-second time. Then when I click to change to a "Scatter plot" - the UI becomes unresponsive. It takes a few minutes to Scatter to visualize anything, and the browser is complaining that "This page is slowing down your computer". 

Notebook:
https://github.com/atoti/notebooks/blob/master/notebooks/ifrs9/main.ipynb

Query:
Opening PD LT on x axes
PD LT on y axes
Color by id

**Comment by tibdex** [comment]
Indeed, I can reproduce this using the app. The pivot table is really fluid but when switching to a scatter plot, the _Google Chrome Helper_ process goes up to 100% CPU usage in the following 30 seconds and Chrome displays a "Page Unresponsive" popup a few seconds later, suggesting to kill the tab.

**Comment by messaooudi** [comment]
The UI team is tracking this request with an [internal ticket](https://support.activeviam.com/jira/browse/UI-5356).

**Comment by bloussou** [comment]
I can reproduce in `0.6.4`

**Comment by tibdex** [comment]
Closing this because the internal ticket has been postponed. The proposed solution is to add filters to your queries to reduce their result size before visualizing them as charts.
